docs/openapi/pn-apikey-manager-external-web.yaml

```diff
@@ -294,6 +294,7 @@ components:
       properties:
         name:
           type: string
+          pattern: '^(?!\s).*(?<!\s)$'
```

Comment:
This regular expression does empty string valid. I should change it with ^(?!\s).+(?<!\s)$, so also the empty string becomes invalid.